.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,7 @@
+- id: swift-format
+  name: swift-format
+  entry: swift-format format --in-place --recursive --parallel
+  language: swift
+  types: [swift]
+  require_serial: true
+ 
```

Comment:
It looks like the file still has a space here on the last line (GitHub is showing the indicator that the file doesn’t have a trailing newline).

Reply:
Okay, yep I've removed the space. Not sure if there is a new trailing line now though.

Reply:
Great. Looks good now.